File: MODUL-4/SOAL 3/PRAK403-2410817120017-RIKAFAULIANARAHMI.py
angka1, angka2 = map(int, input().split())

if angka1 < angka2:
    i, j = angka1, angka2
    while i <= angka2 and j >= angka1:
        print(f"{i} {j}", end=" - " if i < angka2 and j > angka1 else "")
        i += 1
        j -= 1
else:
    i, j = angka1, angka2
    while i >= angka2 and j <= angka1:
        print(f"{i} {j}", end=" - " if i > angka2 and j < angka1 else "")
        i -= 1
        j += 1
print()

File: MODUL-4/SOAL 3/PRAK403-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

void main() {
    int angka1, angka2;

    scanf("%d %d", &angka1, &angka2);

    if (angka1 < angka2) {
        for (int i = angka1, j = angka2; i <= angka2 && j >= angka1; i++, j--) {
            printf("%d %d", i, j);
            if (i < angka2 && j > angka1) { 
                printf(" - ");
            }
        }
    } else {
        for (int i = angka1, j = angka2; i >= angka2 && j <= angka1; i--, j++) {
            printf("%d %d", i, j);
            if (i > angka2 && j < angka1) { 
                printf(" - ");
            }
        }
    }

    printf("\n");
}
